js/start.js

```diff
@@ -62,27 +62,25 @@ function getValidLanguage(lang) {
   return 'en_US';
 }
 
+// Imports specific JSON and Trumbowyg language files for localization.
+function importLanguageFiles(lang) {
+  app.polyglot.extend(require(`./languages/${lang}.json`)); // eslint-disable-line global-require
+
+  const trumboLang = getTrumboLangFileNameByCode(lang);
+  if (trumboLang) {
+    // eslint-disable-next-line global-require
+    app.polyglot.extend(require(`../node_modules/trumbowyg/dist/langs/${trumboLang}.min.js`));
+  }
+}
+
 const initialLang = getValidLanguage(app.localSettings.get('language'));
 app.localSettings.set('language', initialLang);
 moment.locale(initialLang);
 app.polyglot = new Polyglot();
-app.polyglot.extend(require(`./languages/${initialLang}.json`));
-
-let trumboLang = getTrumboLangFileNameByCode(initialLang);
-if (trumboLang) {
-  // eslint-disable-next-line global-require
-  app.polyglot.extend(require(`../node_modules/trumbowyg/dist/langs/${trumboLang}.min.js`));
```

Comment:
@rmisio @jjeffryes: So, this is the specific line I need to change from using polyglot to simply including this specific language file. 

Am I able to programmatically simply change this to:

``` 
require(`../node_modules/trumbowyg/dist/langs/${trumboLang}.min.js`)
```

or is there some other/better way to programmatically inject this trumbo lang file?

Reply:
Well, those language files just add those translations to a `jquery` namespaced variable in the global scope, so in `installRichEditor`, I would do something like:

```
const lang = getTrumboLangFileNameByCode(app.localSettings.get('language'));

if (lang && !window.jQuery.trumbowyg.langs[lang]) {
  require(`../node_modules/trumbowyg/dist/langs/${lang}.min.js`);
}
```